src/CatBox.NET/CatBox.cs

```diff
@@ -1,17 +1,7 @@
 ﻿namespace CatBox.NET;
 
-public interface ICatBox
-{
-    public Task UploadFile();
-    public Task DeleteFile();
-    public Task CreateAlbum();
-    public Task EditAlbum();
-    public Task AddToAlbum();
-    public Task RemoveFromAlbum();
-    public Task DeleteAlbum();
-}
-
-public class Catbox : ICatBox
+[Obsolete("Do not use at this time")]
```

Comment:
Can the `bool` overload for Error be set here to set force a compiler error when attempting to use this class? Or does that have to be set on the constructor?